config/initializers/assets.rb

```diff
@@ -13,5 +13,4 @@
 # Precompile additional assets.
 # application.js, application.css, and all non-JS/CSS in the app/assets
 # folder are already added.
-
-Rails.application.config.assets.precompile += %w( transport_surveys transport_surveys/charts.js )
+Rails.application.config.assets.precompile += %w( transport_surveys.js transport_surveys/carbon.js transport_surveys/charts.js transport_surveys/handlebars_helpers.js transport_surveys/helpers.js transport_surveys/notifier.js transport_surveys/storage.js )
```

Comment:
This seems fine but wondering with the glob if you'd tried something like: 

```
Dir[Rails.root.join('app/assets/javascripts/transport_surveys/*')].map { |f| "transport_surveys/#{File.basename(f)}" }
```

To build the array? Note this only adds the js files within the transport_survey directory, not the main transport_surveys.js file. So will look even uglier to add that, but might be more stable?

Reply:
Thanks Leigh, agree it was messy. Have been reading around and think procs / globs no longer work here with later versions of sprockets, in favour of using manifest.js : https://github.com/rails/sprockets/blob/main/UPGRADING.md#manifestjs

Reply:
See what you think of the latest change anyway... I'm hopeful :-)